source/features/expand-all-hidden-comments.tsx

```diff
@@ -29,6 +26,14 @@ async function handleAltClick({altKey, delegateTarget}: DelegateEvent<MouseEvent
 	}
 }
 
+async function handleAltClick({altKey, delegateTarget}: DelegateEvent<MouseEvent, HTMLButtonElement>): Promise<void> {
+	if (!altKey) {
+		return;
+	}
+
+	await showToast(expandHidden(delegateTarget), {message: 'Expanding…', doneMessage: 'Expanded'});
```

Comment:
Tested, it works wonderfully!


https://user-images.githubusercontent.com/1402241/222095921-5bb61623-75bb-4413-aa1e-726b6be8a039.mov